android/app/src/main/java/com/github/braillesystems/learnbraille/util/AccessibilityHelper.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.braillesystems.learnbraille.util
+
+import android.content.Context
+import android.content.Context.ACCESSIBILITY_SERVICE
+import android.view.accessibility.AccessibilityEvent
+import android.view.accessibility.AccessibilityEvent.TYPE_ANNOUNCEMENT
+import android.view.accessibility.AccessibilityManager
+
+// https://stackoverflow.com/a/50069388 - is there a way to make TalkBack speak?
+class AccessibilityHelper {
+    companion object {
+        @JvmStatic
+        fun announceForAccessibility(context: Context?, announcement: String) {
+            context!!
```

Comment:
I'm not sure whether this line follows best practices (@winter-yuki blamed me the previous time when I used `!!` operator)

I used it because `context` is of type `Context?`

`context` is of type `Context?`, not `Context`, because I pass `this.context` in `AbstractFragmentWithHelp.kt`, and it is of type `Context?`